### PR TITLE
clang-format added and applied

### PR DESCRIPTION
**Pull request by michalbali256**

### CODE REVIEW

File: .clang-format [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  SplitEmptyRecord: false
+  SplitEmptyNamespace: false
+
+BreakBeforeBinaryOperators: NonAssignment
+FixNamespaceComments: true
+
+SortIncludes: true
+IncludeBlocks: Regroup
+IncludeCategories:
+  - Regex:           '<[[:alnum:]._]+>'
```
Comment (asmelko) [inline .clang-format:30]: I think that character `/` should be there as well.

File: .clang-format [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    Priority:        3
+
+IndentCaseLabels: true
+NamespaceIndentation: None
+IndentPPDirectives: AfterHash
+KeepEmptyLinesAtTheStartOfBlocks: false
+MaxEmptyLinesToKeep: 3
+
+SpaceAfterTemplateKeyword: false
+Standard: Cpp11
```
Comment (asmelko) [inline .clang-format:44]: what about Cpp17?

Comment (michalbali256) [reply .clang-format:44]: cpp17 is supported only since clang-format-10 and yields the same results as cpp11.

File: .clang-format (hunk 1 above)
Comment (michalbali256) [reply .clang-format:30]: Is there any include from standard library that has '/' in it?

Comment (asmelko) [reply .clang-format:30]: seems that none

**Pull request merged**